### ISSUE TITLE
Rolling predictions on a test set

### ISSUE THREAD

**Issue by sh0gg0th** [awslabs/gluon-ts#1716]
### Discussed in https://github.com/awslabs/gluon-ts/discussions/1715

<div type='discussions-op-text'>

<sup>Originally posted by **sh0gg0th** October  4, 2021</sup>
Hi all,

I would like to split my time series data into a train and a test set and train a DeepAREstimator on the training set. I would then like to perform sliding window predictions on the test set, e.g. with prediction_length of 4, I would like to be able to take first 4 datapoints t1-t4 from the test set, create predictions for datapoints t5-t8 and calculate the error, then move my window one step to the right, take datapoints t2-t5, predict t6-t9 and so on.

Could you please let me know where to look for an example? Is https://github.com/awslabs/gluon-ts/blob/master/examples/run_rolling_forecast_backtest.py relevant? Many thanks!</div>

**Comment by BEbillionaireUSD** [comment]
Could you please share how did you achieve it? I also want to perform sliding window predictions on a multi-variate test set.
